pkg/controller/unifiedpushserver/postgresql.go

```diff
@@ -181,3 +181,27 @@ func newPostgresqlService(cr *pushv1alpha1.UnifiedPushServer) (*corev1.Service,
 		},
 	}, nil
 }
+
+func newPostgresImageStream(cr *pushv1alpha1.UnifiedPushServer) (*imagev1.ImageStream, error) {
```

Comment:
There should already an ImageStream for PostgreSQL in the `openshift` namespace on OpenShift, so we probably don't need to create our own.

Reply:
I thought about it. 
I guess this depends how the ops manage this situation. Perhaps the Postgres version they want to use for UPS is different than the version they want to use in some other place.
If this is nonsense, I am ok  to drop the ImageStream creation here. Same goes to Oauth Proxy.

Reply:
Sent an email to AeroGear ML about this: https://groups.google.com/forum/#!topic/aerogear/D7vncataJac

Reply:
> I guess this depends how the ops manage this situation. Perhaps the Postgres version they want to use for UPS is different than the version they want to use in some other place.

For our case at least, I believe we'll always want to use an image from the centralized PostgreSQL ImageStream. Different versions can be (and are) specified on that ImageStream with different ImageStreamTags.

Reply:
Yeah, using the OpenShift ImageStream in later commits.